src/services/ipinfo-client.doubles.ts

```diff
@@ -0,0 +1,11 @@
+import { IpinfoClient } from './ipinfo-client';
+import { timeout } from '../utils';
+
+export const fakeResponse = 'AR';
```

Comment:
maybe don't need this constant if only use one time

Reply:
Maybe it's ok to specify that this will be the fake response. To do not return a string that we don't know what is.